lte/gateway/python/magma/mobilityd/ip_allocator_dhcp.py

```diff
@@ -358,10 +351,16 @@ def alloc_ip_address(self, sid: str, vlan: int) -> IPDesc:
 
     def _parse_dhcp_helper_cli_response_to_store(self, dhcp_desc, dhcp_response, mac, vlan):
         try:
-            dhcp_json = json.loads(dhcp_response.stdout)
-        except JSONDecodeError as e:
+            # Only look in at the last line of the stdout for the JSON
+            # Previous lines may contain warnings or other unnecessary info.
+            # This is also helpful for debugging since we can add print
+            # statements to the helper script without breaking the JSON.
```

Comment:
We had some deprecation warnings at some point with a non-standard test setup. I cannot remember the details but it wouldn't be the package version that are installed currently in production. However, it is foreseeable that something like this may eventually become a problem so I think communicating the results via a json file might be smarter.